test/core/testUploaderError.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/* global beforeEach, describe, it */
+
+var expect = require('salinity').expect;
+
+var UploaderError = require('../../lib/core/uploaderError.js');
```

Comment:
Thanks for adding these tests. I was gonna ask for them, but they're already here! \o/ Will look in more detail later, but so far all of this looks pretty good to me.